src/main/java/codeu/controller/UsersServlet.java

```diff
@@ -0,0 +1,42 @@
+package codeu.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import codeu.model.store.basic.UserStore;
+import codeu.model.data.User;
+
+public class UsersServlet extends HttpServlet {
+
+  /**
+   * Store class that gives access to Users.
+   */
+  private UserStore userStore;
```

Comment:
I think we should add a `setUserStore()` so that userStore can be populated. Right now userStore will be null so the username will never be found when you do `User user = userStore.getUser(username);` Also that will help when we go to test.